### ISSUE TITLE
Cover not able to be configured

### ISSUE THREAD

**Issue by joselsegura** [rospogrigio/localtuya#1478]
## The problem
I'm trying to add a cover device and I get this message:

```
Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists.
``` 

## Environment
- Localtuya version: v5.2.1
- Home Assistant Core version: 2023.8.3
- [x] Does the device work using the Home Assistant Tuya Cloud component ?
- [x] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ?
- [] Was the device working with earlier versions of localtuya ? Which one? 
- [] Are you using the Tuya/SmartLife App in parallel ?
- 
## Steps to reproduce
1. Localtuya -> Configure -> Add new device
2. Pick the device from the Discovered devices list
3. Click "Submit" with the default options loaded by Localtuya
4. The error is shown

## DP dump
```
Device 06150873c44f33f710f5 at 192.168.1.39 key 84a1b296b6645a05 protocol 3.3 dev_type type_0a:
    DPS [1] VALUE [3] 
```

**Comment by xZetsubou** [comment]
Provide the logs Settings > System > Logs search for Tuya
Are u running `tinytuya` in the same HA localtuya instance? 
Are you sure that device isn't configured in another HA Instance using local Tuya? or another local integration?
Did you closed Tuya/SmartLife apps before adding the device?

**Comment by xZetsubou** [comment]
> No tinytuya installed, first time configured and no Tuya/SmartLife app running.

Tinytuya wouldn't interrupt with this integration if tiny isn't connected to the device. 
To clarify `your` issue basically there is no detected `DPS`. so whether you are trying to connect using wrong protocol or its not supported eg `3.5` Try using tinytuya to scan the local devices. Scan feature won't works if u run it in the same `localtuya` instance they both uses the same port to scan devices. so run it using another instance.
```
python -m pip install tinytuya
python -m tinytuya scan
```
